modules/rosa-hcp-sts-creating-a-cluster-external-auth-provider-cli.adoc

```diff
@@ -0,0 +1,128 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sts-creating-a-cluster-external-auth-provider-cli_{context}"]
+= Creating an external authentication provider
+:source-highlighter: pygments
+:pygments-style: emacs
+:icons: font
+ 
+After you have created a {hcp-title} cluster with the enabled option for external authentication providers, you must create a provider using the ROSA CLI.
+
+This configuration uses Microsoft Entra ID to authenticate your credentials. The Entra ID values are:
+
+[cols="1,1", options="header"]
+|===
+
+|CLI Flag
+|Expected value
+
+|`--cluster` 
+|This string is the name or the ID of your cluster
+
+|`--name`
+|This string is the name you set for your external authentication provider.
+
+|`--issuer-audiences` 
+|This common-separated list of strings is your client ID. In Microsoft Entra ID, this is called the audience ID.
+
+|`--issuer-url`
+|This string is the URL of the token issuer
+
+|`--claim-mapping-username-claim` 
+|This string is the name of the claim that should be used to construct user names for the cluster identity.
+
+|`--claim-mapping-groups-claim`
+| This string describes rules on how to transform information from an ID token into a cluster identity.
+
+|===
+
+.Procedure
+
+. Use one of the following commands to create your external authentication provider:
+
+* To use the interactive command interface, run the following commands:
++
+[source,terminal]
+----
+$ rosa create external-auth-provider -c <cluster_name>
+----
++
+This command starts an interactive CLI process:
++
+--
+[source,terminal]
+----
+I: Enabling interactive mode
+? Name: <.>
```

Comment:
Is there a reason why you shifted from using hard-coded numbers <1> to the wildcard <.>?